jenkins/values.yaml

```diff
@@ -0,0 +1,219 @@
+# Default values for jenkins.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+## Overrides for generated resource names
+# See templates/_helpers.tpl
+# nameOverride:
+# fullnameOverride:
+
+Nginx:
+  Image: "nginx"
+  ImageTag: "stable-alpine"
+  ImagePullPolicy: "Always"
+  Port: 9090
+  ServiceType: LoadBalancer
+  ServiceAnnotations: {}
+Master:
+  Name: jenkins-master
+  Image: "jenkins/jenkins"
+  ImageTag: "lts"
+  ImagePullPolicy: "Always"
+# ImagePullSecret: jenkins
+  Component: "jenkins-master"
+  UseSecurity: true
+  HostNetworking: false
+  AdminUser: admin
+  # AdminPassword: <defaults to random>
+  resources:
+    requests:
+      cpu: "50m"
+      memory: "256Mi"
```

Comment:
Do we think this is enough memory?

Reply:
Not really. I had to increase memory for the jnlp container to 2048Mi, otherwise it would get OOM sometimes. As with the other case, we can tune it in our values on deployment instead of in the default ones.